src/main/java/com/meta/chatbridge/FBID.java

```diff
@@ -24,7 +24,7 @@ public static FBID from(long value) {
 
   public static FBID from(String value) {
     Objects.requireNonNull(value);
-    long longValue = Long.valueOf(value);
+    long longValue = Long.parseLong(value);
```

Comment:
Don't know the most about Java but this is to return a primitive long value instead of a long object? And valueof has some performance gains from caching?

Reply:
Effective Java, Item 61: Prefer primitive types to boxed primitives